solr/solrj/build.gradle

```diff
@@ -62,6 +62,7 @@ dependencies {
   testImplementation project(':solr:test-framework')
   testImplementation project(':solr:core')
   testImplementation project(':solr:solrj')
+  testRuntimeOnly project(':solr:modules:sql')
```

Comment:
Whenever the solrj-jdbc solrj-module is split out, this will go in there, correct?

So the dependency tree at that point will look like `solr:solrj:jdbc (test)` -> `solr:modules:sql` -> `solr:solrj`

Reply:
@dsmiley had a comment about this on the jira as well regarding moving the solrj tests -> sql module. `JdbcTest` and one of the streamingexpressions jdbc test are what is failing without the `testRuntimeOnly project(':solr:modules:sql')`. I'm not sure it makes sense to move the `solrj` jdbc tests to `solr:modules:sql` since they are really solrj tests. The solrj tests are testing that solrj actually works w/ the SQL module. So I kinda like how the dependencies are defined right now.

Regarding solrj-jdbc module - I would think so unless streaming expressions is somehow still in solrj-core or whatever it will be called.

Reply:
yeah, definitely don't think they should go in the module, definitely leave them where they are for now. I think this is good to go, and the somewhat-recursive looking dependency (though it's test vs src, so not really recursive) will eventually go away.

Reply:
I disagree but I'm not going to hold this up.  If we're adding a dependency between our modules that can be avoided simply by moving a test -- that's a "smell" to me, and the test should move.  I think it's natural that a module might have "solrj level tests" to exercise the backend functional by using higher level APIs... that shouldn't mean (IMO) that SolrJ tests need to depend on nearly everything.  At least that's the direction this will go if we follow your logic.

Reply:
Put differently, if we can make simple moves to reduce intermodule dependencies, even _just_ tests, that's a good thing.

Reply:
I understand your point David, and I think if this was a long term thing I would definitely agree. However I don't particularly mind which solution we go with in this case, since it's really a temporary band-aid until we have the solrj sub-modules, which shouldn't be too far off